st2common/st2common/models/api/base.py

```diff
@@ -283,6 +282,7 @@ def cast_func(value):
             try:
                 result = f(*args, **kwargs)
             except TypeError as e:
+                LOG.debug(traceback.format_exc())
```

Comment:
the stack trace gets swallowed here, i uncovered a bug in the decorator jsexpose function, but nothing critical. I'll remove.